specification/signalr/resource-manager/Microsoft.SignalRService/preview/2021-06-01-preview/signalr.json

```diff
@@ -1576,7 +1579,11 @@
       "properties": {
         "name": {
           "description": "The name of the SKU. Required.\r\n\r\nAllowed values: Standard_S1, Free_F1",
-          "type": "string"
+          "type": "string",
+          "enum": [
+            "Standard_S1",
+            "Free_F1"
```

Comment:
It's not an `enum` in our side. We actually have more values. And it's complicated:
- some values(like `Free_DS2`) are actually supported but they are not encouraged, so not included in description intensionally.
- Another name `Standard_S2` is internal only, it's supported but not exposed to public.

Reply:
Thanks for the information. This change is just a test. Please ignore it.